service/src/main/scala/org/apache/celeborn/server/common/http/api/ApiUtils.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.server.common.http.api
+
+import java.lang.management.{LockInfo, ManagementFactory, MonitorInfo, ThreadInfo}
+
+object ApiUtils {
+  def getThreadDump(): Seq[ThreadStackTrace] = {
+    ManagementFactory.getThreadMXBean.dumpAllThreads(true, true).filter(_ != null).map(
```

Comment:
Because the naming of master/worker threads is unified, could we sort the thread with the order that the thread of master/worker is before other thread, like Spark `Utils#getThreadDump()`?

Reply:
thanks, I am oncall this week, will look this issue later.